acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/steps/TestDataSteps.java

```diff
@@ -43,10 +44,18 @@ public void clearTestData() {
      */
     @After(value = "@DiscardAfter", order = 500)
     public void discardEditedPublication() throws Throwable {
```

Comment:
Could rename to "discardEditedDocument" to keep consistent with other publication to document changes?

Reply:
👍